src/dataprotection/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.11.2'
+VERSION = '0.12.0'
```

Comment:
Please check the version to follow https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md#versioning-practice